crates/matrix-qrcode/Cargo.toml

```diff
@@ -12,14 +12,14 @@ rust-version = "1.56"
 repository = "https://github.com/matrix-org/matrix-rust-sdk"
 
 [package.metadata.docs.rs]
-features = ["docs"]
-rustdoc-args = ["--cfg", "feature=\"docs\""]
+features = ["docsrs"]
```

Comment:
Wouldn't `all-features = true` be easier for the crates that don't have mutually-exclusive features?

Reply:
Eh, maybe. I always fear that we'll have some feature that we would like to have hidden.

Reply:
Well, at least for some crates `all-features` makes sense. Indeed.